src/nft/utils/calcPoolPrice.ts

```diff
@@ -21,7 +21,7 @@ export const calcPoolPrice = (asset: GenieAsset, position = 0) => {
       : (asset.sellorders[0] as Deprecated_SellOrder)
 
   const decimals = BigNumber.from(1).mul(10).pow(18)
-  const ammFee = nft.ammFeePercent ? (100 + (nft.ammFeePercent as number)) * 100 : 110 * 100
+  const ammFee = nft?.ammFeePercent ? (100 + (nft.ammFeePercent as number)) * 100 : 110 * 100
```

Comment:
how did this previously pass typescript lint if it can be undefined?

Reply:
and is the cast to number necessary?

Reply:
Previously we only called calcPoolPrice on pooled assets or assets from the v3 sweep endpoint that had a placeholder ammFeePercent field. The change to gql no longer provided that field.